packages/nextjs/pages/[contractAddress]/[network].tsx

```diff
@@ -103,30 +120,34 @@ const ContractDetailPage = ({ addressFromUrl, chainIdFromUrl }: ServerSideProps)
       const fetchContractAbi = async () => {
         setIsLoading(true);
 
-        if (storedAbi && storedAbi.length > 0) {
-          setContractData({ abi: storedAbi, address: contractAddress });
+        const storedAbi = getAbiFromLocalStorage(contractAddress as Address, parsedNetworkId);
+        if (storedAbi) {
+          setContractData({ abi: storedAbi, address: contractAddress as Address });
```

Comment:
Umm why are doing unnecessary assertion (`as Address`) everywhere in this PR 😅

We have already typed `Address: string` so I think in most place in this PR its not required, could you please remove it from other place as well(not from all places just saying for files changed in this PR wherever possible)

Reply:
Thanks for reminding me I totally forgot about these lol